wandb/sdk/artifacts/artifact.py

```diff
@@ -2199,6 +2044,11 @@ def logged_by(self) -> Optional[Run]:
         )
 
     def json_encode(self) -> Dict[str, Any]:
+        """
+        Description.
```

Comment:
Description.

Reply:
Returns the artifact encoded as a json of the following form:
{
        "_type": "artifactVersion",
        "_version": "v<version_index>",
        "id": artifact_id,
        "version": source_artifact_version,
        "sequenceName": source_artifact_name,
        "usedAs": "<name override in a run using the artifact>",
    }